modules/sdk/project-templates/project-templates-form-field/src/main/resources/archetype-resources/src/main/resources/META-INF/resources/__artifactId__Register.soy

```diff
@@ -0,0 +1,6 @@
+#parse ("definitions.vm")
+{namespace ${className}Register}
+
+{deltemplate PageRenderer.RegisterFieldType variant="'${formFieldTypeName}'"}
+    {call ${className}.render data="all" /}
+{/deltemplate}
```

Comment:
@gabrielibson, on your commit message, you say that you're updating the `build.gradle`, but where is this update on this commit?